### ISSUE TITLE
EC platform scale and stability - 10 parallel experiments

### ISSUE THREAD

**Issue by may-lim** [everycure-org/matrix#499]
**Aim:**

1. Our platform enables people to execute the pipeline E2E in the cloud without needing help from an engineering colleague by submitting the pipeline from their machines to the cluster.
2. We can host many experiments in parallel
3. Scaling these is solved given current pipeline design

**Description:**

Target stability to accelerate our speed of learning, unblock modelling team, and free up EC Engineering capacity

**Tasks which need to be completed:**
@pascalwhoop which tasks do you think are critical for this for Oct?
- [x] #456 
- [x] #77 
- [x] #348 
- [ ] Conversations/pair programming driven by EC Engineering with subs on the way we work together

**This Sprint Goal is completed when:**
 
Critical tasks identified above have PRs merged into main
